CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
+* enable decimal for Protobuf ([#6884](https://github.com/confluentinc/ksql/pull/6884)) ([6b5877c](https://github.com/confluentinc/ksql/commit/6b5877c2910023263421b247824af4eadb7db640))
+* Make pull queries streamed asynchronously ([#6813](https://github.com/confluentinc/ksql/pull/6813)) ([b69e3f8](https://github.com/confluentinc/ksql/commit/b69e3f8e459c3d0a055ec0738ae71e58bda7b28c))
+* make UDAFs configurable and remove limit on COLLECT_LIST/SET ([#6851](https://github.com/confluentinc/ksql/pull/6851)) ([63ae169](https://github.com/confluentinc/ksql/commit/63ae169ab753aab980f0ee4cbf52fbe34291418d))
+* Move WHERE clause handling for pull queries into logical plan ([#6814](https://github.com/confluentinc/ksql/pull/6814)) ([fa6758a](https://github.com/confluentinc/ksql/commit/fa6758af2f46594bfdf8f18a6d59a9b40e62e1d0))
+* new and initialize commands for migration tool ([#6988](https://github.com/confluentinc/ksql/pull/6988)) ([8bdb09a](https://github.com/confluentinc/ksql/commit/8bdb09a2751365966cf5a4a1436276d7a3e0d9e1))
+* Rewrites pull query WHERE clause to be in DNF and allow more expressions  ([#6874](https://github.com/confluentinc/ksql/pull/6874)) ([b8e0c99](https://github.com/confluentinc/ksql/commit/b8e0c991a982229afd6d42dc8752a29a35fa937e))
+* Support timestamp protobuf serde ([#6927](https://github.com/confluentinc/ksql/pull/6927)) ([5ea1ce4](https://github.com/confluentinc/ksql/commit/5ea1ce4c9129260272608aa8d8263e22a23a9188))
+* timestamp support - casting, comparisons and serde ([#6806](https://github.com/confluentinc/ksql/pull/6806)) ([a27df46](https://github.com/confluentinc/ksql/commit/a27df465040bf3aa409ca34509c0117fadf8982a))
+
+
+
+### Bug Fixes
+
+* Changes a couple query validation tests after upgrade to antlr ([#6941](https://github.com/confluentinc/ksql/pull/6941)) ([96288e4](https://github.com/confluentinc/ksql/commit/96288e454fc06a6f84193d771f7139b43a985e77))
+* Changes a query validation tests after upgrade to antlr ([#6976](https://github.com/confluentinc/ksql/pull/6976)) ([e635489](https://github.com/confluentinc/ksql/commit/e635489630a6c2a5f804231a88bf79218ca57932))
+* client/server SSL settings fail when 'ssl.key.password' is set ([#6763](https://github.com/confluentinc/ksql/pull/6763)) ([3e48540](https://github.com/confluentinc/ksql/commit/3e48540fbe1a02ed02faa667e424b225f9d7ba20))
+* do not log data on JSON deserialization errors (MINOR) ([#6930](https://github.com/confluentinc/ksql/pull/6930)) ([1834da6](https://github.com/confluentinc/ksql/commit/1834da678cf86e7cb333d57a9c97d1884d09d7af))
+* dont use the streamthread to construct the log-metric-and-continue logger ([#6909](https://github.com/confluentinc/ksql/pull/6909)) ([6af3bc6](https://github.com/confluentinc/ksql/commit/6af3bc6fc49de977c5571bb0af5188ac186516e9))

```

Comment:
This commit actually made it into 0.15 but due to a bug in the changelog generation script is appearing in the auto-generated 0.16 changelog. It should actually be in the 0.15 changelog but appears to be missing since the 0.15 changelog was auto-generated. Mind moving this to the 0.15 section as part of this PR?